frontend/src/app/helpers/common.const.ts

```diff
@@ -11,6 +15,24 @@ export const COMMON = {
     questionType: {
         shortAnswer: 'Short Answer',
         multipleChoice: 'Multiple Choice'
+    },
+    securityQuestionSet1: {
+        q1: 'In what city or town was your first job?',
```

Comment:
I don't think we need any other constant variables further than these. I left these in here since I thought it would be a bit messy having constant strings/variables in other files and we could just change one file in case we need to. Feel free to suggest other things and make changes